comp-sci/data-structures-and-algorithms/intro-data-structures/the-stack-data-structure.md

```diff
@@ -33,7 +33,7 @@ A **stack** is a collection of objects optimized for two things:
 
 The items are inserted and removed according to the *last-in, first-out* (*LIFO*) principle.
 
-![stack](https://img.enkipro.com/9a05488e0127141d09e7bf0d600f8407.png)
+![stack](https://img.enkipro.com/df45877c1de4e1891f45d451089edc3e.png)
```

Comment:
background inside squares shouldn't be orange.

here is how the image looked before https://img.enkipro.com/9a05488e0127141d09e7bf0d600f8407.png